lib/grizzly/zwave/commands/schedule_entry_lock_time_offset_report.ex

```diff
@@ -0,0 +1,73 @@
+defmodule Grizzly.ZWave.Commands.ScheduleEntryLockTimeOffsetReport do
+  @moduledoc """
+  This command is used to advertise the time zone offset and daylight savings parameters.
+
+  Params:
+
+    * `:sign_TZO` - Plus (0) or minus (1) sign to indicate a positive or negative offset from UTC.
+
+    * `:hour_TZO` - Specify the number of hours that the originating time zone deviates from UTC.
+
+    * `:minute_TZO` - Specify the number of minutes that the originating time zone deviates UTC.
+
+    * `:sign_offset_DST` - Plus (0) or minus (1) sign to indicate a positive or negative offset from UTC.
+
+    * `:minute_offset_DST` - This field MUST specify the number of minutes the time is to be adjusted when daylight savings mode is enabled.
+
+  """
+
+  @behaviour Grizzly.ZWave.Command
+
+  alias Grizzly.ZWave.{Command, DecodeError}
+  alias Grizzly.ZWave.CommandClasses.ScheduleEntryLock
+
+  @type param ::
+          {:sign_TZO, 0 | 1}
```

Comment:
{:sign_tzo, :minus | :plus} and encoding/decoding changes from this. Also all lowercase to be consistent with other Grizzly commands.